Dockerfile

```diff
@@ -23,13 +21,18 @@ COPY webhooks/ webhooks/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-FROM $image 
+FROM gcr.io/distroless/static:nonroot as release
 WORKDIR /
 COPY --from=builder /workspace/manager .
 USER $user:$user
```

Comment:
Given that `user` is defaulted to `root`, I'm pretty sure that we *don't* want this. Definitely not for the distroless release image (where we're explicitly trying to run as non-root), and probably not for the debug image either.

Can we remove this line altogether for the release image? Do we need it for the debug image?

Reply:
👍, removed the user arg and changed it to `nonroot`.

Since the debug image was previously set to run as root, I've removed the user argument entirely.

Reply:
:+1: I believe the image defaults to `nonroot` anyway, but setting it explicitly isn't harmful.